src/platforms/salesforce/schemas.ts

```diff
@@ -325,6 +332,89 @@ export const salesforceNoteUpdate = validators.object({
     .partial(),
 });
 
+// -
+// ContentDocumentLink (used for ContentNote)
+// -
+export const salesforceContentDocumentLink = z
+  .object({
+    Id: z.string(),
+    ContentDocumentId: z.string(),
+    LinkedEntityId: z.string(),
+    IsDeleted: z.boolean(),
+  })
+  .partial();
+
+export const salesforceContentDocumentLinkCreate = validators.object({
+  ContentDocumentLink: z.object({
+    ContentDocumentId: z.string(),
+    LinkedEntityId: z.string(),
+  }),
+});
+
+export const salesforceContentDocumentLinkCreateResponse = validators.object({
+  id: z.string(),
+});
+
+// -
+// ContentNoteContent
+// -
+export const salesforceContentNoteContent = validators.object({
+  body: z.object({
+    _readableState: z.any(),
+  }),
+});
+
+// -
+// ContentNote
+// -
+export const salesforceContentNote = validators
+  .object({
+    Id: z.string(),
+    CreatedDate: validators.date(),
+    LastModifiedDate: validators.date(),
+    TextPreview: z.string(),
+    Content: z.string(),
```

Comment:
just to be clear - this is a URL right?

Reply:
Yep, I can add a comment just to make that explicit